maps/map_files/BigRed/standalone/crashlanding-eva.dmm

```diff
@@ -433,12 +408,6 @@
 	icon_state = "wy15"
 	},
 /area/bigredv2/outside/general_offices)
-"bJ" = (
-/obj/item/ammo_magazine/rifle/heap{
```

Comment:
Uneccesary removal of a lore item, HEAP is unobtainable in normal gameplay so an empty mag shouldn't be an issue in staying.